hack/local-up-kubeedge.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+KUBEEDGE_ROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..
+ENABLE_DAEMON=${ENABLE_DAEMON:-false}
+LOG_DIR=${LOG_DIR:-"/tmp"}
+
+
+if [[ "${CLUSTER_NAME}x" == "x" ]];then
+    CLUSTER_NAME="test"
+fi
+
+export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
+
+# spin up cluster with kind command
+function kind-up-cluster {
```

Comment:
make all function names with "_" instead of  "-".

Reply:
done.